packages/now-next/src/index.ts

```diff
@@ -1454,24 +1456,32 @@ export const build = async ({
             message: 'invariant: htmlFsRef != null && jsonFsRef != null',
           });
         }
+
+        if (!canUsePreviewMode) {
+          htmlFsRef.contentType = htmlContentType;
+          prerenders[outputPathPage] = htmlFsRef;
+          prerenders[outputPathData] = jsonFsRef;
+        }
       }
 
-      prerenders[outputPathPage] = new Prerender({
-        expiration: initialRevalidate,
-        lambda,
-        fallback: htmlFsRef,
-        group: prerenderGroup,
-        bypassToken: prerenderManifest.bypassToken,
-      });
-      prerenders[outputPathData] = new Prerender({
-        expiration: initialRevalidate,
-        lambda,
-        fallback: jsonFsRef,
-        group: prerenderGroup,
-        bypassToken: prerenderManifest.bypassToken,
-      });
+      if (prerenders[outputPathPage] == null) {
```

Comment:
Should this check be loose or can we just check for a truthy value like `if (prerenderes[outuptPathPage])`?

Reply:
I think this if clause is fine as-is, really. All of them accomplish the same thing, the current one isn't "loose" imo. Only stricter would be `=== undefined`.